cobc/typeck.c

```diff
@@ -3103,7 +3103,7 @@ cb_build_const_start (struct cb_field *f, cb_tree x)
 		if (cb_field_variable_size (p)) {
 			cb_error (_("variable length item not allowed here"));
 			return cb_build_numeric_literal (0, "1", 0);
-		}
+	}
```

Comment:
This looks wrong (same above) - broken indent upstream that will be fixed with one of the next merges? If those fixes are in a commit easy to apply, please do so.

Reply:
Indeed. This was introduced in 3.x by 5112, and as of HEAD, it has not been fixed...

I can make a quick commit in 3.x to fix that (direct to SVN) and merge it here.
(I believe this does not require a ChangeLog entry)

Reply:
agreed